server/models/migrations/next/migrate.sql

```diff
@@ -13,6 +13,7 @@ ALTER TABLE `debtor_group` ADD COLUMN `is_non_client_debtor_groups` TINYINT(1) N
  * @description: Budget Report #7683
  */
 INSERT INTO unit values (320, 'Budget Report','TREE.BUDGET_REPORT','',281,'/reports/budget_report');
+<<<<<<< HEAD
```

Comment:
@lomamech , looks like this is a broken merge.   Please remove the merge related code.